packages/next/pages/_document.tsx

```diff
@@ -81,7 +81,7 @@ function getPreNextScripts(context: HtmlProps, props: OriginProps) {
         <script
           {...scriptProps}
           key={scriptProps.src || index}
-          defer={!disableOptimizedLoading}
+          defer={scriptProps.defer ?? !disableOptimizedLoading}
```

Comment:
@janicklas-ralph can you review this. It looks fine to me.

Reply:
@timneutkens Is there any chance to get this moving forward? We can't migrate to Script components because of this issue.

Reply:
This looks good to me. We can merge this as soon as possible.